examples/traverse-ipld-graphs/create-node.js

```diff
@@ -4,7 +4,8 @@ const IPFS = require('../../src/core')
 // In your project, replace by the following line and install IPFS as a dep
 // const IPFS = require('ipfs')
 
-function createNode (options = {}) {
+function createNode (options) {
+  options = options || {}
```

Comment:
If you call it as `createNode(null)` (as some of our tests do) you get surprising results.  I went over the code and tried to standardise it on `options || {}` instead of `(options = {})`